cli/templates/package.json.hbs

```diff
@@ -16,7 +16,7 @@
   "license": "{{project.license}}",
 {{/if}}
   "dependencies": {
```

Comment:
Not part of this change, but shouldn't this be in `devDependencies`? I think most neon projects would be top level applications and not libraries.

Reply:
As long as `neon build` is needed for running the build, it has to be in `"dependencies"`. If we can get rid of `neon build` (which would be awesome!) then we can move it to `"devDependencies"`.